lib/models/api/collection/create_request.dart

```diff
@@ -45,7 +45,7 @@ class CreateRequest {
     map['keyDecryptionNonce'] = keyDecryptionNonce;
     map['encryptedName'] = encryptedName;
     map['nameDecryptionNonce'] = nameDecryptionNonce;
-    map['type'] = type;
+    map['type'] = type.toString();
```

Comment:
`toString` will still return the same since it's not overridden? Might have to call `typeToString(type)` instead